multiuser/machine-auth/che-multiuser-machine-authentication/src/main/java/org/eclipse/che/multiuser/machine/authentication/server/MachineTokenRegistry.java

```diff
@@ -96,6 +96,26 @@ public String getUserId(String token) throws NotFoundException {
   }
 
   /**
+   * Gets workspaceId by machine token
+   *
+   * @return workspace identifier
+   * @throws NotFoundException when no such machine token exists
+   */
+  public String getWorkspaceId(String token) throws NotFoundException {
```

Comment:
this method should be tested in multiuser/machine-auth/che-multiuser-machine-authentication/src/test/java/org/eclipse/che/multiuser/machine/authentication/server/MachineTokenRegistryTest.java

Reply:
Can't find where this method is used

Reply:
This method is used by the `rh-che` assembly, in order to get an *up-to-date* Keycloak token that we can inject into the Bayesian Language server when starting it.

See PR https://github.com/redhat-developer/rh-che/pull/417, and especially [this line](https://github.com/redhat-developer/rh-che/pull/417/files#diff-7b667d7b1bc0a0182258f5095b7d3411R43)